biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -994,7 +1020,20 @@ subroutine fuse_cohorts(currentSite, currentPatch, bc_in)
 
                  !Criteria used to divide up the height continuum into different cohorts.
 
-                 if (diff < dynamic_fusion_tolerance) then
+                 if (diff < dynamic_size_fusion_tolerance) then
+
+                    ! Only fuse if the cohorts are within x years of each other 
+                    ! if they are the same age we make diff 0- to avoid errors divding by zero
+                    !NB if cohort age tracking is off then the age of both should be 0
+                    ! and hence the age fusion criterion is met    
+                   if (currentCohort%coage .eq. nextc%coage) then
```

Comment:
might recommend the following logic here:

if( abs(currentCohort%coage-nextc%coage)<nearzero) then

Reply:
Sounds good, I've changed it.